pywinauto/windows/uia_element_info.py

```diff
@@ -55,6 +56,28 @@ def elements_from_uia_array(ptrs, cache_enable=False):
     return elements
 
 
+def is_element_satisfies_criteria(element, process=None, class_name=None, name=None, control_type=None,
+                                  content_only=None, **kwargs):
+    """Check if element satisfy filter criteria"""
+    is_appropriate_control_type = True
+    if control_type:
+        if isinstance(control_type, string_types):
+            is_appropriate_control_type = element.CurrentControlType == IUIA().known_control_types[control_type]
+        elif not isinstance(control_type, int):
```

Comment:
If in element_info.py above already changed to ` if isinstance(depth, integer_types) `, shouldn't use the same check against `six.integer_types`?

Reply:
Change in element_info.py is caused by a failed test on python 2.7. I agree, it's useful to make same changes here just in case.